### ISSUE TITLE
Font issue on Debian Buster (no characters in "open file" window)

### ISSUE THREAD

**Issue by rauferd** [senshu/Sozi#396]
When opening Sozi 17.02.072116-linux-x64 on my Debian 10 (buster) 64bit system (by executing `./Sozi-17.02.072116-linux-x64/Sozi`), the "main" window and the "open file" window appear more or less simultaneously.
In the "open file" window, no characters are printed, but just empty boxes where the characters should be. There is no error message; neither visually in the GUI nor on the console. There is one empty box per character, which led me to the conclusion that this may be a font issue.

I read that Droid Sans needs to be available for compiling Sozi, so I installed that font systemwide. I can use it now in LibreOffice and the like, but no luck in Sozi.
The following files are available in the subfolder `resources/app/vendor/DroidSans/`, and - just to be safe - I also put them in `vendor/DroidSans/`:
```
DroidSans-Bold.ttf
DroidSans.ttf
DroidSans-Bold.eot
DroidSans.eot
```
 Still no luck.

I can do some debugging, but I have no idea where to start.

P.S.: I can manage to open a file even without the labels in the "open file" dialog. In the "main" window, both the UI and the SVG contents are rendered correctly; no missing or awkward fonts here.

**Comment by senshu** [comment]
From your report, I understand that the problem concerns only the file chooser dialog, and that the text of the main UI is displayed correctly.

I did not fully understand if you only used the official build of Sozi 17.02, or if you also tried to build it by yourself.
In the first case, the font is bundled with the application and you don't need to install it.

I advise you to try Sozi 17.06 for two reasons:

* In Sozi 17.06, pressing F12 shows the developer tools. Relevant error messages will be found in the JavaScript error console.
* Sozi 17.06 uses a more recent version of [electron](https://electron.atom.io/). Other users have reported that it solved some UI issues.

**Comment by rauferd** [comment]
I can confirm that this is resolved in Sozi 17.06.
Thank you for keeping up the great work!
